packages/client/src/components/forum-topic/forum-topic.tsx

```diff
@@ -0,0 +1,51 @@
+import { ForumTopicFeedItem } from '@/components';
+import { Button, Form } from 'antd';
+import { Input } from '@/components';
+import './forum-topic.css';
+
+interface Message {
+  messageId: number;
+  name: string;
+  text: string;
+  time: string;
+}
+
+interface ForumTopicProps {
+  title: string;
+  description: string;
+  feed: Message[];
+}
+
+export const ForumTopic = ({
+  title,
+  description,
+  feed,
+}: ForumTopicProps): JSX.Element => {
+  return (
+    <div className='forum-topic'>
+      <h1>{title}</h1>
+      <div className='forum-topic__description'>{description}</div>
+      {feed.length ? (
+        <div className='forum-topic__feed'>
+          {feed.map(item => (
+            <ForumTopicFeedItem
+              key={item.messageId}
+              name={item.name}
+              text={item.text}
+              time={item.time}
+            />
+          ))}
+        </div>
+      ) : null}
```

Comment:
[Предложение]
Так как при `feed.length = 0` будет отображаться `null`, можно заменить тернарный оператор на логическое И. Так будет проще читать эту запись. только нужно будет сделать условие, со значением `boolean` - `feed.length > 0`, чтобы 0 не отрисовался
```suggestion
      {feed.length > 0 && (
        <div className='forum-topic__feed'>
          {feed.map(item => (
            <ForumTopicFeedItem
              key={item.messageId}
              name={item.name}
              text={item.text}
              time={item.time}
            />
          ))}
        </div>
      )}
```

Reply:
Спасибо, заменил.